jekyll/_cci2/server/v4.3/installation/upgrade-server.adoc

```diff
@@ -18,6 +18,31 @@ A successful deployment will update the web app. Unless noted in the release not
 
 NOTE: We recommend that you do not skip releases when upgrading.
 
+NOTE: Related to new execution systems
+* Docker Layer Caching (DLC)
+** old docker layer caching (dlc) volumes will not be carried over and need to be wiped manually
+** dlc usage delete threshold is currently not configurable at this time
+** dlc prune days is currently not configurable at this time
+** dlc volume size is currently not configurable at this time
+** dlc wont get cleaned up on s3 (projects that stop using dlc wont delete final DLC cache without manual intervention)
```

Comment:
```suggestion
** projects that stop using dlc wont delete final DLC cache without manual intervention
```
I'm not sure the first part is clear, it kind of suggests that the s3 bucket will just infinitely grow. The bit in brackets explains the situation great I think